src/batterystatus.cpp

```diff
@@ -39,12 +39,26 @@
 #include <mce/dbus-names.h>
 #include <mce/mode-names.h>
 
+namespace {
+const QString MceSettingsChargingMode = "/system/osso/dsm/charging/charging_mode";
+const QString MceSettingsChargingLimitEnable = "/system/osso/dsm/charging/limit_enable";
+const QString MceSettingsChargingLimitDisable = "/system/osso/dsm/charging/limit_disable";
```

Comment:
```
const auto MceSettingsChargingLimitDisable = QStringLiteral("/system/osso/dsm/charging/limit_disable");
```
I tend to use QStringLiterals for this kind of constants. Woboq guys wrote a good article back in time: https://woboq.com/blog/qstringliteral.html

Reply:
Fixed